spec/std/base64_spec.cr

```diff
@@ -130,6 +142,11 @@ describe "Base64" do
         Base64.decode_string("a")
       end
     end
+
+    it "decode bug" do
```

Comment:
Could the example description be more *descriptive*? What are the conditions that lead to the bug? Perhaps stating *it decodes <certain condition>* makes more sense.